### ISSUE TITLE
Applied plugin class names are not decipherable in verbose mode

### ISSUE THREAD

**Issue by oalders** [houseabsolute/perl-code-tidyall#75]
This is running under `prove`, in parallel and without the verbose flag, but I'm not sure that's relevant here.

`# [applying the following plugins: Code::TidyAll::Plugin::PerlTidy=HASH(0x3d74030)]`

See https://travis-ci.org/metacpan/metacpan-web/jobs/231876490#L636

**Comment by autarch** [comment]
Are you saying you just want the class name?

**Comment by oalders** [comment]
Yeah, I guess I'm saying that the `=HASH(0x3d74030` makes it hard to see what's going on when scanning a test log.

**Comment by autarch** [comment]
I agree. A patch would be welcome. Note that sometimes plugins have a name beyond just the class name in the config file, so we should use that when it's set.

**Comment by autarch** [comment]
Hmm, looking at the code, it should already be showing a useful name, so this is some sort of bug.
